src/controller/SetUpCodePairer.cpp

```diff
@@ -553,7 +563,6 @@ void SetUpCodePairer::StopAllDiscoveryAttempts()
     LogErrorOnFailure(StopDiscoveryOverBLE());
     LogErrorOnFailure(StopDiscoveryOverDNSSD());
     LogErrorOnFailure(StopDiscoveryOverSoftAP());
-    LogErrorOnFailure(StopDiscoveryOverWiFiPAF());
```

Comment:
We really need WiFi PAF to support two separate things:

1. Stop trying to find new things to connect to (but don't touch existing connections).
2. Shut down any connections we have.

The first one of those should happen here.  The second of those should happen wherever we need to do that.

Doing `StopDiscoveryOverWiFiPAF()` only in `StopPairing` is broken: that only gets called during controller shutdown or due to someone calling StopPairing on the controller, but we want StopDiscoveryOverWiFiPAF to happen when things time out and pairing shuts _itself_ down.